app/src/main/res/values/strings.xml

```diff
@@ -197,6 +199,11 @@
     <string name="app_settings_screen_title">App Settings</string>
     <string name="give_feedback_screen_title">Give Feedback</string>
     <string name="report_bug_screen_title">Report Bug</string>
+    <string name="about_app_screen_title">About This App</string>
+    <string name="settings_privacy_policy_label">Privacy Policy</string>
+    <string name="settings_terms_of_use_label">Terms of Use</string>
+    <string name="label_copyright">Copyright</string>
+    <string name="label_copyright_value">© Wire Swiss GmbH</string>
```

Comment:
Maybe:

```suggestion
    <string translatable="false" name="label_copyright_value">© Wire Swiss GmbH</string>
```

Reply:
Yes! Good catch, also was not really sure if we should have it just in the `strings` file or something else